### PR TITLE
PSP-3436 Search by Legal description

### PR DESCRIPTION
**Pull request by asanchezr**
screenshot

### CODE REVIEW

File: frontend/public/index.html [hunk 1]
```diff
@@ -9,7 +9,7 @@
...
+      content="default-src 'self'; frame-ancestors 'self'; img-src 'self' blob: data: https://dev-motigeo.th.gov.bc.ca/ https://openmaps.gov.bc.ca/ https://maps.gov.bc.ca/ https://server.arcgisonline.com/; frame-src 'self' https://*.oidc.gov.bc.ca https://oidc.gov.bc.ca http://localhost:8080; connect-src 'self' https://dev-motigeo.th.gov.bc.ca/ https://*.oidc.gov.bc.ca/ https://oidc.gov.bc.ca http://localhost:8080/ https://openmaps.gov.bc.ca/ https://*.apps.gov.bc.ca/ https://apps.gov.bc.ca/ https://server.arcgisonline.com/; form-action 'self'; style-src 'self' 'unsafe-hashes' 'unsafe-inline'; script-src 'self' 'sha256-yt+SNVxRkIi6H6yb7ndFuZM1esMX9esg3UpRHaTsyVk=';"
```
[the remark this answers is not in the document]
Comment (devinleighsmith) [reply frontend/public/index.html:12]: Also if you could make yourself a note to release this in TEST in addition to your normal release steps.

File: frontend/src/components/common/List/SelectInput.tsx [hunk 2]
```diff
@@ -3,14 +3,23 @@ import { getIn, useFormikContext } from 'formik';
...
+  /** Specifies that the HTML element should be disabled */
   disabled?: boolean;
+  /** The option to show on the drop-down by default */
   defaultKey?: keyof SelectTypes;
+  /** The field name */
   field: string;
+  /** The underlying HTML element to use when rendering the FormControl */
+  as?: React.ElementType;
+  /** optional help text to display below the FormControl */
+  helpText?: string;
```
Comment (devinleighsmith) [inline frontend/src/components/common/List/SelectInput.tsx:15]: I see a helpText prop here, but it doesn't seem to be used.

File: frontend/src/components/common/form/InputGroup.tsx [hunk 3]
```diff
@@ -18,6 +18,8 @@ type OptionalAttributes = {
   label?: string;
   /** The underlying HTML element to use when rendering the FormControl */
   as?: React.ElementType;
+  /** optional help text to display below the FormControl */
+  helpText?: string;
```
Comment (devinleighsmith) [inline frontend/src/components/common/form/InputGroup.tsx:22]: is this getting passed through to the underlying input?

File: frontend/public/index.html (hunk 1 above)
[the remark this answers is not in the document]
Comment (asanchezr) [reply frontend/public/index.html:12]: Yes, thanks for the reminder. Was wondering where to document this.

File: frontend/src/components/common/List/SelectInput.tsx (hunk 2 above)
Comment (asanchezr) [reply frontend/src/components/common/List/SelectInput.tsx:15]: It is a passthrough all the way down to the **Input** (passing thru **InputGroup**)

File: frontend/src/components/common/form/InputGroup.tsx (hunk 3 above)
Comment (asanchezr) [reply frontend/src/components/common/form/InputGroup.tsx:22]: you got it - see my previous comment

File: frontend/public/tenants/tenant.json [hunk 4]
```diff
@@ -12,5 +12,9 @@
     "body": ""
   },
   "layers": [],
-  "propertiesUrl": "ogs-internal/ows?service=wfs&request=GetFeature&typeName=PIMS_PROPERTY_LOCATION_VW&outputformat=json&srsName=EPSG:4326&version=2.0.0&"
+  "propertiesUrl": "ogs-internal/ows?service=wfs&request=GetFeature&typeName=PIMS_PROPERTY_LOCATION_VW&outputformat=json&srsName=EPSG:4326&version=2.0.0&",
+  "parcelMapFullyAttributed": {
+    "url": "ogs-internal/ows",
+    "name": "PMBC_PARCEL_POLYGON_FABRIC"
+  }
```
Comment (asanchezr) [inline frontend/public/tenants/tenant.json:19]: New configuration entry on **tenant.json** - will update https://confluence.th.gov.bc.ca/display/PIMS/2.1 with deployment instructions

File: frontend/src/components/common/form/Input.tsx [hunk 5]
```diff
@@ -129,6 +132,7 @@ export const Input: React.FC<InputProps> = ({
           value={pattern ? restricted : rest.value ?? value}
           title={pattern ? restricted : rest.value ?? value}
           placeholder={placeholder}
+          aria-describedby={helpText ? `${field}-help-text` : undefined}
```
Comment (asanchezr) [inline frontend/src/components/common/form/Input.tsx:135]: improving component accessibility (ie screen readers) when help text is provided

File: frontend/src/components/maps/leaflet/mapUtils.tsx [hunk 6]
```diff
@@ -155,18 +155,30 @@ export const asProperty = (point: PointFeature): IProperty => {
...
+  const cqlValue: string = toCqlFilterValue(object, pidOverride);
+  return cqlValue.length ? `cql_filter=${encodeURIComponent(cqlValue)}` : '';
+};
+
+/**
+ * Convert any object to a cql filter string value, assuming the object's keys should be used as CQL filter properties.
+ * NOTE: The resulting string value is left as-is. It will not be URL-encoded
+ * @param object an object to convert to a cql filter string.
+ */
+export const toCqlFilterValue = (object: any, pidOverride?: boolean) => {
```
Comment (asanchezr) [inline frontend/src/components/maps/leaflet/mapUtils.tsx:172]: Needed to call this function to get the cql filter value without `cql_filter={value}`

**Review by devinleighsmith — APPROVED** [review]

File: frontend/src/mocks/mockFAParcelLayerResponse.ts [hunk 7]
```diff
@@ -0,0 +1,72 @@
+export const mockFAParcelLayerResponse = {
```
[the remark this answers is not in the document]
Comment (asanchezr) [reply frontend/src/mocks/mockFAParcelLayerResponse.ts:1]: right - let's talk about this on our dev meeting on Friday. Would love to see it changed across for all mocks

**Pull request merged**